docs/src/main/sphinx/optimizer/pushdown.rst

```diff
@@ -151,6 +151,65 @@ to perform the remaining query processing on a smaller amount of data.
 The specifics for the supported pushdown of table joins varies for each data
 source, and therefore for each connector.
 
+However, there are some generic conditions that must be met in order for a join
+to be pushed down:
+
+* join pushdown must be enabled
```

Comment:
```suggestion
```

it's enabled by default, so this sentence is misleading -- user doesn't need to enable anything

while "join pushdown must not be disabled" would be correct thing to write, it would sound a bit silly to me. Maybe we need that still?

Reply:
I believe join pushdown is disabled by default based on https://github.com/trinodb/trino/blob/ab9f02df0f5c2bf22ba12be6678c371dd91fa435/plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/JdbcMetadataConfig.java#L28-L34

This also aligns with what I have observed while using the feature in Trino 359.

Reply:
> I believe join pushdown is disabled by default based on

you're correct.